project/en-US/stm/stm.xml

```diff
@@ -3,11 +3,77 @@
+        from JTA?" We will try to answer those and similar questions:
+        <orderedlist>
+                <listitem>
+                    <para>
+                        The goal of STM is to simplify object reads and writes from multiple threads/protect state from
+                        concurrent updates. The Narayana STM implementation will safely manage any conflicts between these
+                        threads using whatever isolation model has been chosen to protect that specific state instance. In STM,
+                        there are two isolation implementations, pessimistic (the
+                        default), which would cause conflicting threads to be blocked until the original has completed its
+                        updates (committed or aborted the transaction); then there is the optimistic approach which allows
+                        all of the threads to proceed and checks for conflicts at commit time, where one or more of the threads
+                        may be forced to abort if there have been conflicting updates.
+                    </para>
+                </listitem>
+                <listitem>
+                    <para>
+                        STM objects have state but it does not need to be persistent (durable). In fact the default behaviour
+                        is for objects managed within transactional memory to be volatile, such that if the service or
+                        microservice within which they are being used crashes or is spawned elsewhere, e.g., by a scheduler,
+                        all state in memory is lost and the objects start from scratch. But surely you get this and more with
+                        JTA (and a suitable transactional datastore) and do not need to worry about restarting your application?
+                        Not quite. There is a trade-off here: we are doing away with persistent state and the overhead of reading
+                        from and then writing (and sync-ing) to the datastore during each transaction. This makes updates to
+                        (volatile) state very fast but you still get the benefits of atomic updates across multiple STM objects
+                        (e.g., objects your team wrote then calling objects you inherited from another team and requiring them
+                        to make all-or-nothing updates), as well as consistency and isolation in the presence of concurrent
+                        threads/users (common in distributed microservices architectures). Furthermore, not all stateful
+                        applications need to be durable - even when JTA transactions are used, they tend to be the exception
+                        and not the rule.
+                    </para>
+                </listitem>
+                <listitem>
+                    <para>
+                        Another benefit of STM is composability and modularity. You can write concurrent Quarkus objects/services
```

Comment:
I think this is the first mention of Quarkus in our manual. Perhaps it could be hyperlinked to https://quarkus.io/ or some other more appropriate resource for finding further information?

Reply:
It shouldn't be in there. I'll remove it.